File: data/models_typescript/UserSignedUp.ts

class UserSignedUp {
  private _email?: string;

  constructor(input: {
    email?: string,
  }) {
    this._email = input.email;
  }

  get email(): string | undefined { return this._email; }
  set email(email: string | undefined) { this._email = email; }
}
export default UserSignedUp;

File: data/models_python/UserSignedUp.py
from pydantic import BaseModel as PydanticBaseModel, Field
from typing import *

class BaseModel(PydanticBaseModel):
    class Config:
        allow_population_by_field_name = True

from typing import *

class UserSignedUp(BaseModel):
  email: Optional[str] = Field(alias='Email of the user')
